gtd-timer/Controllers/LogInController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+using gtdtimer.Services;
+using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DTO;
+
+namespace gtdtimer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class LogInController : ControllerBase
+    {
+        private ApplicationUserManager<User> userManager;
+        private ILogInService logInService;
+        public LogInController(ApplicationUserManager<User> userManager, LogInService logInService)
+        {
+            this.userManager = userManager;
+            this.logInService = logInService;
+        }
+
+        [HttpPost]
+        [Route("login")]
+        public async Task<IActionResult> Login([FromBody] LoginDTO model)
+        {
+            if (model == null)
+            {
+                return BadRequest("Request is Null");
+            }
+            //After creating UserManager this next line will uncomented
```

Comment:
never comments like this in code that goes to pull request / review: if some component is missing or not ready yet - create ticket to track it

Reply:
in fact - all of this should be moved to service